Makefile

```diff
@@ -24,20 +24,33 @@ validate:
 	./hack/verify-generated-docs.sh
 
 .PHONY: build
-build: build-windows build-linux build-darwin
+build: build-windows build-linux build-darwin build-windows-arm build-linux-arm build-darwin-arm
 
 .PHONY: build-windows
 build-windows:
 	CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -o ./bin/kyma.exe $(FLAGS) ./cmd
+	
+.PHONY: build-windows-arm
+build-windows-arm:
+	CGO_ENABLED=0 GOOS=windows GOARCH=arm go build -o ./bin/kyma.exe $(FLAGS) ./cmd
 
 .PHONY: build-linux
 build-linux:
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./bin/kyma-linux $(FLAGS) ./cmd
 
+.PHONY: build-linux-arm
+build-linux-arm:
+	CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -o ./bin/kyma-linux-arm $(FLAGS) ./cmd
+
 .PHONY: build-darwin
 build-darwin:
 	CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build -o ./bin/kyma-darwin $(FLAGS) ./cmd
 
+.PHONY: build-darwin-arm
+build-darwin-arm:
+	go version | grep '1.16'
```

Comment:
Hi @Shanduur, the build for ARM on Mac OSX is not working because of this grep. Our CI/CD is running on go 1.15.7 and thus it fails.

Is ARM support for OSX only available on g 1.16?

Reply:
Unfortunately on Go <1.16 outside MacOS this will result with:
```
/home/user/.go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/tmp/go-link-271834784/go.o: file not recognized: file format not recognized
collect2: error: ld returned 1 exit status

make: *** [Makefile:54: build-darwin-arm] Error 2
```

Reply:
As discussed, ARM support on Mac OSX is commented out until our CI/CD runs on go 1.16.